From 4948c51f958bf422de62e1e111bad1d0cf4e0b2a Mon Sep 17 00:00:00 2001
From: SpiritCroc <dev@spiritcroc.de>
Date: Tue, 11 Sep 2018 17:38:04 +0200
Subject: [PATCH 13/13] SoundTriggerHalLegacy.cpp: include errno.h

frameworks/av/services/soundtrigger/SoundTriggerHalLegacy.cpp:72:17: error: use of undeclared identifier 'ENODEV'
        return -ENODEV;
                ^
frameworks/av/services/soundtrigger/SoundTriggerHalLegacy.cpp:83:17: error: use of undeclared identifier 'ENODEV'
        return -ENODEV;
                ^
frameworks/av/services/soundtrigger/SoundTriggerHalLegacy.cpp:91:17: error: use of undeclared identifier 'ENODEV'
        return -ENODEV;
                ^
frameworks/av/services/soundtrigger/SoundTriggerHalLegacy.cpp:102:17: error: use of undeclared identifier 'ENODEV'
        return -ENODEV;
                ^
frameworks/av/services/soundtrigger/SoundTriggerHalLegacy.cpp:110:17: error: use of undeclared identifier 'ENODEV'
        return -ENODEV;
                ^
frameworks/av/services/soundtrigger/SoundTriggerHalLegacy.cpp:118:17: error: use of undeclared identifier 'ENODEV'
        return -ENODEV;
                ^
frameworks/av/services/soundtrigger/SoundTriggerHalLegacy.cpp:124:13: error: use of undeclared identifier 'ENOSYS'
    return -ENOSYS;
            ^

Change-Id: I3bd324de2e73a757d654d0f255873cfc7cc064e7
---
 services/soundtrigger/SoundTriggerHalLegacy.cpp | 1 +
 1 file changed, 1 insertion(+)

diff --git a/services/soundtrigger/SoundTriggerHalLegacy.cpp b/services/soundtrigger/SoundTriggerHalLegacy.cpp
index 2b78818a2..573c74590 100644
--- a/services/soundtrigger/SoundTriggerHalLegacy.cpp
+++ b/services/soundtrigger/SoundTriggerHalLegacy.cpp
@@ -16,6 +16,7 @@
 
 #include <utils/Log.h>
 #include "SoundTriggerHalLegacy.h"
+#include <errno.h>
 
 namespace android {
 
-- 
2.17.1

